presto-iceberg/src/test/java/com/facebook/presto/iceberg/procedure/TestIcebergRegisterProcedure.java

```diff
@@ -373,6 +400,17 @@ public void testUnregisterTable()
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testUnregisterTableWithNamedArguments()
+    {
+        String tableName = "unregister";
+        assertUpdate("CREATE TABLE " + tableName + " (id integer, value integer)");
+
+        // Unregister table with procedure
+        assertUpdate("CALL system.unregister_table(table_name => '" + tableName + "', schema => '" + TEST_SCHEMA + "')");
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+    }
+
```

Comment:
Would it be better to rename either the this test class or Add a new Test class to incorporate `unregister_table` procedure related tests?

Also test classes for `SetTablePropertyProcedure` & `RollbackToSnapshotProcedure` procedures are missing, should we add few tests for that as well Or may be handle tests in future PR? WDYT?

Reply:
> Would it be better to rename either the this test class or Add a new Test class to incorporate unregister_table procedure related tests?

Good suggestion, I have renamed the test class.

> Also test classes for SetTablePropertyProcedure & RollbackToSnapshotProcedure procedures are missing, should we add few tests for that as well Or may be handle tests in future PR?

Under your reminder, I realized that there are still some different kinds of work to be done. I think it's better to leave the work of registering and documenting `SetTablePropertyProcedure`, adding test cases for both `SetTablePropertyProcedure` and `RollbackToSnapshotProcedure` to subsequent separated PRs. And in this PR, we focus on support named arguments uniformly for existing procedures, and unifying the naming conventions for their arguments. Do you think this makes sense?

Reply:
Sounds good to me 👍🏻